src/mkdocs_spellcheck/plugin.py

```diff
@@ -66,6 +67,7 @@ class SpellCheckPlugin(BasePlugin):
         ("backends", MkType(list, default=["symspellpy"])),
         ("known_words", MkType((str, list), default=[])),
         ("skip_files", MkType(list, default=[])),
+        ("skip_file_globs", MkType(list, default=[])),
```

Comment:
Can't we simply merge the glob feature into the existing `skip_files` option?

Reply:
That should work. I will see if I can get to it today.

Reply:
I combined the options.